CoreFoundation/String.subproj/CFString.c

```diff
@@ -1476,16 +1476,18 @@ CF_PRIVATE CFStringRef __CFStringCreateImmutableFunnel3(
                 useNullByte = hasNullByte;
 
             } else {	// Inline data; reserve space for it
-
+            
```

Comment:
This is bleeding whitespace that wasn't there before. Please be careful to revert the commit so that you don't unintentionally introduce such changes.

Reply:
It appears this bleeding whitespace is back. Please revert.